Sources/Sentry/Public/SentryDefines.h

```diff
@@ -61,6 +61,16 @@ typedef void (^SentryOnCrashedLastRunCallback)(SentryEvent *_Nonnull event);
  */
 typedef BOOL (^SentryShouldQueueEvent)(
     NSHTTPURLResponse *_Nullable response, NSError *_Nullable error);
+
+/**
+ * Function pointer for a sampler callback.
+ *
+ * @param samplingContext context of the sampling.
+ *
+ * @return A sample rate between 0.0 and 1.0.
```

Comment:
explain in the comment that 0.0 means, don't capture anything, 1.0 captures everything

Reply:
Agree. Also nice to throw in a: 0.01 captures 1% of transactions.